src/arrayMethodJoin.js

```diff
@@ -4,8 +4,23 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let joinedString = '';
+    const glue = separator === null ? 'null' : separator;
```

Comment:
I think this line `const glue = separator === null ? 'null' : separator;`
is unnecessary because you have already set a default value for the separator above)